src/cf-core/src/cf-base.less

```diff
@@ -291,6 +291,14 @@ h4,
     .h6 + & {
         margin-top: unit( 30px / @font-size, em );
     }
+
+    .respond-to-max( @bp-xs-max, {
+        @h4-font-size-on-xs: 16px;
+
+        margin-bottom: unit( 10 / @h4-font-size-on-xs, em );
+        font-size: unit( @h4-font-size-on-xs / @base-font-size-px, em );
+        line-height: unit( 18 / @h4-font-size-on-xs );
```

Comment:
Not sure there's a need for `unit()` here, would `18 / @h4-font-size-on-xs` have the same output?

Reply:
Another case where I should've looked at prior art :) I'll find out.

Reply:
It looks like we need the `unit` to ensure any units in the component parts of a calculation get dropped, not carried through to the result. Your suggestion would result in `line-height: 1.125px;`.

Reply:
Ahhh, I thought Less dropped units if only one value had them. Thanks for checking in on it.